parametric/protos/apm_test_client.proto

```diff
@@ -10,6 +10,18 @@ service APMClient {
   rpc InjectHeaders(InjectHeadersArgs) returns (InjectHeadersReturn) {}
   rpc FlushSpans(FlushSpansArgs) returns (FlushSpansReturn) {}
   rpc FlushTraceStats(FlushTraceStatsArgs) returns (FlushTraceStatsReturn) {}
+
+  rpc OtelStartSpan(OtelStartSpanArgs) returns (OtelStartSpanReturn) {}
+  rpc OtelEndSpan(OtelEndSpanArgs) returns (OtelEndSpanReturn) {}
+  rpc OtelIsRecording(OtelIsRecordingArgs) returns (OtelIsRecordingReturn) {}
+  rpc OtelSpanContext(OtelSpanContextArgs) returns (OtelSpanContextReturn) {}
+  rpc OtelSetStatus(OtelSetStatusArgs) returns (OtelSetStatusReturn) {}
+  rpc OtelSetName(OtelSetNameArgs) returns (OtelSetNameReturn) {}
+  rpc OtelSetAttributes(OtelSetAttributesArgs) returns (OtelSetAttributesReturn) {}
+  rpc OtelFlushSpans(OtelFlushSpansArgs) returns (OtelFlushSpansReturn) {}
+  rpc OtelFlushTraceStats(OtelFlushTraceStatsArgs) returns (OtelFlushTraceStatsReturn) {}
```

Comment:
Are flush calls specific OTel?

Reply:
Flush method is explicit in [the api](https://opentelemetry.io/docs/reference/specification/trace/sdk/#forceflush)

Reply:
It is for spans. So does the traces stats flush has to be OTel specific?

Reply:
Since Force Flushing is defined in the [otel sdk](https://opentelemetry.io/docs/reference/specification/trace/sdk/#forceflush) and not the [otel api](https://opentelemetry.io/docs/reference/specification/trace/api/#tracerprovider), the python implementation does not expose an api for flushing otel traces. This functionality is hidden in a private method but can be accessed by tests. I believe the only TracerProvider operation  we need to support is ‘getTracer()’ and the otel tracer only needs to support creating/activating spans. 

Is forced flushing an operation we would like to expose to otel users or is it something that’s only required in tests?